includes/GlobalFunctions.php

```diff
@@ -2913,6 +2913,8 @@ function wfShellExec( $cmd, &$retval = null, $environ = array() ) {
 
 	wfInitShellLocale();
 
+	wfRunHooks( 'wfShellExecBeforeEnviron', [ &$cmd, &$environ ] ); // Wikia change
```

Comment:
1. AFAIR it's usually `BeforeSomething` which I'd like to see to be followed here.
2. I guess the `Environ` suffix is supposed to note that you'll get environment in the arguments - I'd skip it - looking at the line with `wfRunHooks` makes it obvious it's getting passed to handlers